docs/tutorials/customizing-the-output/ingress-annotator.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: default
-title: "Ingress annotator"
-permalink: /tutorials/migration-workflow/ingress-annotator
-parent: "Migration workflow"
+title: "Adding annotations to Ingress"
```

Comment:
Do we want to call the task as adding annotations to ingress or 'adding custom annotations to kubernetes yamls' and in the contents we can state it as for example ingress

Reply:
done